### DISCUSSION TITLE
Couldn't find lyrics-finder

### DISCUSSION THREAD

**Discussion by BlueMaxWasTaken** [eritislami/evobot#249] [Q&A]
When I use node index.js for the first time, command prompt said cannot find module lyrics-finder. I looked through the files and yes, there are no lyrics-finder. How can I fix this?

**Comment by badRyuk** [comment]
Have you installed `lyrics-finder`. If you haven't use `npm i lyrics-finder`. Or else search the `package.json` file for lyrics-finder. This occurs if you don't have a required package.

**Comment by BlueMaxWasTaken** [comment]
> Have you installed `lyrics-finder`. If you haven't use `npm i lyrics-finder`. Or else search the `package.json` file for lyrics-finder. This occurs if you don't have a required package.

Hello badRyuk, after I done that it works. But now I want to get a soundcloud clientid. But I can't seem to find it. Can you help me?

**Comment by badRyuk** [comment]
Oh for the SoundCloud Client ID. There is a guide link present in  README.MD. Looks like you didn't see it. Anyways here is the guide to getting the SoundCloud ID. Visit **[SoundClioud ID Guide](https://github.com/zackradisic/node-soundcloud-downloader#client-id)**. I used this and I was easily able to get it. If you still can't I will try to further help you out. Sorry for the late reply.
